<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableExecs"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>* Double-click Name to open, and double-click Status to download if indicated.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>** These executables are custom-built and may contain errors. Please back up your project and use them at your own risk</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>724</width>
     <height>21</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="addNew"/>
    <addaction name="addPy"/>
    <addaction name="separator"/>
    <addaction name="configPath"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="addNew">
   <property name="icon">
    <iconset theme="window-new"/>
   </property>
   <property name="text">
    <string>Add exe</string>
   </property>
  </action>
  <action name="configPath">
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
  <action name="addPy">
   <property name="text">
    <string>Add py</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
